.gitignore

```diff
@@ -4,6 +4,9 @@
 # VS Code stuff
 .vscode
 
+# Manifest toml from project toml
+**/Manifest.toml
```

Comment:
Keep in mind that the Manifest needs to be checked in for this branch to work.

Reply:
You need the Manifest in order to depend on the local Wflow, rather than the published one.
It is also a good idea to check it in, in order to keep builds reproducible and synchronized between machines.

Reply:
I see! But if I look at the current checked in manifests, they do not point to the local Wflow version, but to the released version. Is that correct?